GenderPayGap.WebUI/Views/ReportOverview/ReportOverview.cshtml

```diff
@@ -1,257 +0,0 @@
-@using GenderPayGap.Core

```

Comment:
There are other files that should be deleted as well - `ReportOverviewModel` and all the task files

Reply:
Do you mean the TaskList files?